package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "http-headers",
+  "name": "@readme/http-headers",
   "version": "1.0.0",
   "description": "Retrieve HTTP Header Descriptions ",
   "main": "dist/index.js",
   "type": "module",
```

Comment:
I'm guessing this is not supposed to be ESM, right?

```suggestion
```

Reply:
It is, unfortunately this package is `esm` only, as the sum of its parts.